### ISSUE TITLE
Extension does not work in `Viewer` distribution

### ISSUE THREAD

**Issue by tony2y** [bpmn-io/diagram-js-grid#2]
### What I tried

```
import GridModule from 'diagram-js-grid';
import ZoomScrollModule from "diagram-js/lib/navigation/zoomscroll";
import MoveCanvasModule from "diagram-js/lib/navigation/movecanvas";
import BpmnViewer from "bpmn-js/lib/Viewer";

  this.bpmnViewer = new BpmnViewer({
            container: this.$refs.flowCanvas,
            height: 'calc(100vh - 200px)',
            additionalModules:[
              ZoomScrollModule,
              MoveCanvasModule,
              GridModule
            ]
          });
```

### What I expected

The grid shows (as shown in the repository screenshot).

### What happens instead

The grid does not show.

**Comment by nikku** [comment]
Exactly!

**Comment by tony2y** [comment]
I mean, it doesn't work to write in the above way. You can't use gridModule！

@nikku

**Comment by tony2y** [comment]
<img width="731" alt="image" src="https://github.com/bpmn-io/diagram-js-grid/assets/38598582/9c17dde6-17f2-416d-84d0-05af45c64737">
The background is blank and not used diagram-js-grid，How can it be displayed？

**Comment by nikku** [comment]
@smbea Can you have a look at this issue and confirm or deny this is a bug?

**Comment by tony2y** [comment]
@smbea @nikku
I need help !!!  can you help me?

**Comment by smbea** [comment]
I'm checking this now

**Comment by smbea** [comment]
I can confirm this doesn't work with the viewer

**Comment by tony2y** [comment]
Thank you very much. If you find the reason, please tell me. I really need this feature. Thanks again! ! ! @smbea

**Comment by nikku** [comment]
The underlying issue is that this library does not show the grid per default, but only when instructed to do so (which happens via `gridSnapping`, a modeling module).

I propose we change the default to `true` so whenever you pull in this library it will show the grid. We accept contributions to fix this.

**Comment by nikku** [comment]
Fixed via [`v1.0.0`](https://github.com/bpmn-io/diagram-js-grid/releases/tag/v1.0.0).
